rasa/cli/data.py

```diff
@@ -336,6 +342,50 @@ def _convert_nlg_data(args: argparse.Namespace) -> None:
         )
 
 
+def _migrate_responses(args: argparse.Namespace) -> None:
+    """Migrate retrieval intent responses to the new 2.0 format.
+
+    It does so modifying the stories and domain files.
+    """
+    if args.format == "yaml":
+        _migrate_responses_in_domain(args)
+        rasa.utils.common.run_in_loop(
+            _convert_to_yaml(args, StoryResponsePrefixConverter())
+        )
+        telemetry.track_data_convert(args.format, "responses")
+    else:
+        rasa.shared.utils.cli.print_error_and_exit(
+            "Could not recognize output format. Supported output formats: "
+            "'yaml'. Specify the desired output format with '--format'."
+        )
+
+
+def _migrate_responses_in_domain(args: argparse.Namespace):
```

Comment:
Perfectly it would be awesome to convert the domain using `TrainingDataConverter` interface as well 😬 
In order to do that we need to pass `args.out` and `args.data` (`args.domain` for domain) directly to the `_convert_to_yaml()`, and the rest will be taken care of!

Do you still have capacity to do that?

Reply:
done in e908f5a2868 ✅ One question: is it ok to keep the name `TrainingData` in `TrainingDataConverter` now that it can also take care of the domain? I think it's fine, just wanted to double check

Reply:
I think it's ok! The only alternative I can think about is `ProjectData`, but that doesn't look any better 😬